app/src/main/java/org/groundplatform/android/AbstractActivity.kt

```diff
@@ -23,12 +24,18 @@ import androidx.appcompat.app.AppCompatActivity
 import androidx.core.view.ViewCompat
 import androidx.core.view.WindowInsetsCompat
 import javax.annotation.OverridingMethodsMustInvokeSuper
+import org.groundplatform.android.ui.settings.onAttach
 import org.groundplatform.android.util.Debug
 import org.groundplatform.android.util.systemInsets
 
 /** Base activity class containing common helper methods. */
 abstract class AbstractActivity : AppCompatActivity() {
 
+  override fun attachBaseContext(newBase: Context) {
+    val context = onAttach(newBase)
+    super.attachBaseContext(context)
+  }
```

Comment:
Why do we need this change both here and in the application class? Is only the application class change sufficient?